pkg: Enable to capture data for only host processes. (#1741)
inspektor-gadget/inspektor-gadget: docs/ig.md

  }
]
```

For example, with `--host`, you can get the following output:

```bash
$ sudo ig trace exec --host
CONTAINER                                               PID        PPID       COMM             RET ARGS

# Open another terminal.

> $ ls > /dev/null

Reviewer: I suggest to use a different command in the container and outside to make it easier to understand the `ig` output.

```suggestion
$ cat /dev/null
```

$ docker run --name test-host --rm -t debian sh -c 'ls > /dev/null'
# Go back to first terminal.
CONTAINER                                               PID        PPID       COMM             RET ARGS
                               5671             4040             ls               0   /usr/bin/cat /dev/null
test-host                      5757             5735             sh               0   /bin/sh -c ls > /dev/null

> test-host                      5778             5757             ls               0   /bin/ls

Reviewer 2: Please fix the aligment of the columns header and the content

```

> As you can see, events generated from containers have their container field set, while events which are generated from the host do not.

Reviewer 2: ```suggestion
Events generated from containers have their container field set, while events which are generated from the host do not.
```

> As you can see, events generated from containers have their container field set, while events which are generated from root mount namespace do not.

Reviewer: ```suggestion
As you can see, events generated from containers have their container field set, while events which are generated from the host do not.
```

I would avoid saying "root mount namespace" because that's also true for other mount namespaces as long as they are not from containers.
